### PR TITLE
Create Invite struct and InviteCreateJson struct

### PR DESCRIPTION
**Pull request by DistortedPumpkin**

### CODE REVIEW

**Comment by hydrostaticcog** [comment]
@DistortedPumpkin just run `cargo fmt`

**Review by hydrostaticcog — APPROVED** [review]

**Comment by tazz4843** [comment]
We're currently having a very long discussion on Discord as to why he can't run `cargo fmt`.

**Comment by hydrostaticcog** [comment]
I see

File: src/types/invites.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ///
+    /// 128 bit unsigned integer
+    pub owner_id: u128,
+    
+    /// The invite's guild ID
+    ///
+    /// 128 bit unsigned integer
+    pub guild_id: u128,
+
+    /// When the invite was created as a timezone-naive
```
Comment (tazz4843) [inline src/types/invites.rs:16]: Did you mean
> timezone-naive datetime
```suggestion
    /// When the invite was created as a timezone-naive datetime
```

Comment (DistortedPumpkin) [reply src/types/invites.rs]: yep i did, i was writing the line, and then forgot lol. lemme edit

**Review by Cryptex-github — APPROVED** [review]

**Review by tazz4843 — APPROVED** [review]

**Pull request merged**